package.json

```diff
@@ -115,13 +115,13 @@
         "liquidjs": "^10.7.0",
         "mitt": "~3.0.1",
         "mongodb": "~4.17.1",
-        "mqtt": "~4.3.7",
+        "mqtt": "~5.3.5",
```

Comment:
You are doing a major version bump here with a lot of intermediate versions.

Have you carefully read through the changelog? Have you tested that this feature still works?